src/bufio.c

```diff
@@ -630,8 +632,10 @@ application code does not crash during writes to a broken pipe.
       stream->type = BUFIO_PIPE;  // TODO: Restructure code
       if (stream->mode & O_WRONLY) {
         stream->fd = STDOUT_FILENO;  // Write-only
+        fcntl(stream->fd, F_SETFL, (long)(O_WRONLY | O_NONBLOCK));
```

Comment:
According to all manpages I was trying to find, the `arg` used to with `cmd` `F_SETFL` should be of type `int`.

Reply:
Thanks, fixed.